<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>550</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(216, 117, 196, 255), stop:1 rgba(255, 255, 255, 255))
}</string>
   </property>
   <widget class="QPushButton" name="applicationscreen">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>390</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="backscreen">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>508</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Back to Menu</string>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="malevoice">
    <property name="geometry">
     <rect>
      <x>488</x>
      <y>424</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Read Text</string>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="detailsscreen">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>468</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Details</string>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_1">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>360</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>360</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_1">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>8</y>
      <width>330</width>
      <height>350</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>352</x>
      <y>9</y>
      <width>330</width>
      <height>350</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>264</x>
      <y>469</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Clear Screen</string>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>389</y>
      <width>200</width>
      <height>161</height>
     </rect>
    </property>
    <widget class="QPushButton" name="writerscreen">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Note Pad</string>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>200</width>
       <height>161</height>
      </rect>
     </property>
     <widget class="QPushButton" name="writerscreen_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Note Pad</string>
      </property>
      <property name="iconSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>488</x>
      <y>421</y>
      <width>200</width>
      <height>89</height>
     </rect>
    </property>
    <widget class="QPushButton" name="femalevoice">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>48</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Read Translation</string>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>264</x>
      <y>424</y>
      <width>200</width>
      <height>105</height>
     </rect>
    </property>
    <widget class="QPushButton" name="translateButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Translate Text</string>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>223</x>
      <y>392</y>
      <width>25</width>
      <height>153</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>464</x>
      <y>392</y>
      <width>25</width>
      <height>153</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <zorder>widget_4</zorder>
   <zorder>widget_3</zorder>
   <zorder>widget_2</zorder>
   <zorder>applicationscreen</zorder>
   <zorder>backscreen</zorder>
   <zorder>malevoice</zorder>
   <zorder>detailsscreen</zorder>
   <zorder>comboBox_1</zorder>
   <zorder>comboBox_2</zorder>
   <zorder>textEdit_1</zorder>
   <zorder>textEdit_2</zorder>
   <zorder>clearButton</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
